specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2022-08-11-preview/HybridCompute.json

```diff
@@ -0,0 +1,1637 @@
+        },
+        "adFqdn": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the AD fully qualified display name."
+        },
+        "dnsFqdn": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the DNS fully qualified display name."
+        },
+        "privateLinkScopeResourceId": {
+          "type": "string",
+          "description": "The resource id of the private link scope this machine is assigned to, if any."
+        },
+        "parentClusterResourceId": {
+          "type": "string",
+          "description": "The resource id of the parent cluster (Azure HCI) this machine is assigned to, if any."
+        },
+        "mssqlDiscovered": {
+          "type": "string",
+          "description": "Specifies whether any MS SQL instance is discovered on the machine."
+        },
+        "detectedProperties": {
+          "$ref": "#/definitions/DetectedProperties",
+          "description": "Detected properties from the machine."
+        }
+      },
+      "description": "Describes the properties of a hybrid machine."
+    },
+    "MachineUpdateProperties": {
+      "type": "object",
+      "properties": {
+        "locationData": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/locationData"
+        },
+        "osProfile": {
+          "$ref": "#/definitions/OSProfile"
+        },
+        "cloudMetadata": {
+          "$ref": "#/definitions/CloudMetadata",
+          "description": "The metadata of the cloud environment (Azure/GCP/AWS/OCI...)."
+        },
+        "parentClusterResourceId": {
+          "type": "string",
+          "description": "The resource id of the parent cluster (Azure HCI) this machine is assigned to, if any."
+        },
+        "privateLinkScopeResourceId": {
+          "type": "string",
+          "description": "The resource id of the private link scope this machine is assigned to, if any."
+        }
+      },
+      "description": "Describes the ARM updatable properties of a hybrid machine."
+    },
+    "Machine": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/MachineProperties",
+          "description": "Hybrid Compute Machine properties"
+        },
+        "resources": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MachineExtension"
+          },
+          "description": "The list of extensions affiliated to the machine"
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/Identity"
+        },
+        "systemData": {
+          "readOnly": true,
+          "description": "The system meta data relating to this resource.",
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData"
```

Comment:
>systemData [](http://example.com/codeflow?start=94&length=10)

systemdata is already included in the TrackedResource deifnition

Reply:
Hi @rkmanda. These are not related to our current set of updates to the spec.

We can take these as items for our next version update.

Currently delayed due to the slow progress of this spec review.